fabric-entity-attribute-api-v1/src/main/java/net/fabricmc/fabric/api/entity/attribute/v1/FabricDefaultAttributeRegistry.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.entity.attribute.v1;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import net.minecraft.entity.EntityType;
+import net.minecraft.entity.LivingEntity;
+import net.minecraft.entity.attribute.DefaultAttributeContainer;
+import net.minecraft.util.registry.Registry;
+
+import net.fabricmc.fabric.mixin.entity.attribute.DefaultAttributeRegistryAccessor;
+
+/**
+ * Allows registering custom default attributes for living entities.
+ *
+ * <p>All living entity types must have default attributes registered.</p>
+ *
+ * <p>A registered default attribute for an entity type can be retrieved through
+ * {@link net.minecraft.entity.attribute.DefaultAttributeRegistry#get(EntityType)}.</p>
+ *
+ * <p>This API is <strong>experimental</strong> and subject to breaking changes without
+ * prior deprecation or warning.</p>
+ *
+ * @see net.minecraft.entity.attribute.DefaultAttributeRegistry
+ * @deprecated Vanilla snapshot feature, subject to vanilla change; experimental API
```

Comment:
The message doesn't match the standard form, see https://github.com/FabricMC/fabric/issues/499